test/controllers/charge_controller_create_test.js

```diff
@@ -0,0 +1,199 @@
+'use strict'
+
+// Core dependencies
+const path = require('path')
+
+// NPM dependencies
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const expect = require('chai').expect
+
+const chargeId = '42mdrsshtsk4chpeoifhlgf4lk'
+
+const card = {
```

Comment:
As with the comment below, we should use `payment_fixtures`#`validCardDetails` to construct this JSON